doc/configuration.rst

```diff
@@ -1549,8 +1549,12 @@ are:
 
 In order to enable JupyterLite links with Sphinx-Gallery, you need to install
 the `jupyterlite-sphinx <https://jupyterlite-sphinx.readthedocs.io>`_ package.
-For `jupyterlite-sphinx>=0.8` (released 15 March 2023) you also need to install
-`jupyterlite-pyodide-kernel`.
+Versions of `jupyterlite-sphinx` and `sphinx-gallery` should be compatible, but
```

Comment:
```suggestion
Versions of ``jupyterlite-sphinx`` and Sphinx-Gallery should be compatible, but
```

Reply:
I don't quite follow the meaning here. Do you mean that the user should select versions of `jupyterlite-sphinx` and SG that are compatible or that all versions of the two packages are compatible?

Reply:
Thanks for raising it – I think what I wish to convey here is that recent versions for both packages should work fine. How should we better phrase it?